goal_src/jak2/engine/process-drawable/process-drawable.gc

```diff
@@ -603,8 +603,6 @@
       )
     )
   :code (behavior ((arg0 string))
-    ;; og:preserve-this
-    (format 0 "ART ERROR: ~A for ~A~%" arg0 self)
```

Comment:
Do you think any of the remaining art errors are worth looking into? I think a lot of them are original game errors at this point.

Reply:
Nope. They're either missing art data from the original game (unfixable without some new hacks) or extremely rare GOAL bugs.